capitals.rb

```diff
@@ -150,4 +149,60 @@
 }, {
     name: "Wyoming",
     capital: "Cheyenne"
+
 }]
+
+states = states.shuffle
+
+puts "Welcome to 50 states trivia. See how well you can match states and capitals!"
+
+loop do
+  states.each do |state|
+    state[:correct] = 0
+    state[:wrong] = 0
+    i = 0
+
+puts "What is the capital of #{state[:name]}?"
```

Comment:
Make sure to properly indent your code. It's good practice to do this as you go, but something that might help if things get out-of-hand is Atom's auto-indent feature (Edit -> Lines -> Auto Indent).